### PR TITLE
Legibility + design improvs to jump-to link section inside main article

### PR DESCRIPTION
**Pull request by klepas**
## Description
- wrapped the `article`’s `h1` and jumpto link section in a `header`
- padded the new `article` page header from the `article` content
- padded the jump to list items from each other
- switched from flex boxes to CSS columns
- switched from `ul` to `ol`
## Additional information

Screengrabs from before–after:

before-desktop-jumpto

before-mbl-jumpto

after-desktop-jumpto

after-mbl-jumpto
## Questions/discussion

The styling for this currently sits inside `kss.scss`; we should lift out generic jumpto/ToC styling into the ui-kit. Any thoughts?

### CODE REVIEW

**Comment by klepas** [comment]
@hannah-ustwo, @TrebBrennan, @petronbot please review and discuss (:

**Comment by klepas** [comment]
Ohyea, and we prolly want to add some JS to make the link scrolling smooth out, as noted in standup.

**Comment by TrebBrennan** [comment]
This looks kind of broken when there is a single orphan list item.
Maybe we should just do a single list which is closer to Agency's & ATO's design?
list
screen-shot-2016-07-07-at-3 23 47-pm

**Comment by klepas** [comment]
Known issue: there is a leakage of margins across the 2 CSS columns which screws up the 2nd column if there are an odd # of items... fine in gecko but screwed up in webkit.

Reference: https://stackoverflow.com/questions/5314726/css-multi-column-layout-of-list-items-doesnt-align-properly-in-chrome

**Comment by klepas** [comment]
@TrebBrennan Yea, Petra and I just tried wrangling this with no luck.

**Comment by TrebBrennan** [comment]
**Simpler, Clearer, and Faster.** Lets opt for a single traditional list?
1) Without number I'm not sure of the reading order for these columns.
2) It'll also help with someone using a magnifier because they don't need to scroll around the place to find the content.

**Comment by klepas** [comment]
@TrebBrennan yea, definitely can see the attraction of a single-column.

I’ve made a range of changes and improvements — keen to have these reviewed in light of the chatter on moving back to one column.

**Comment by hannah-ustwo** [comment]
@TrebBrennan we were conversing about this over here just now too, about how we can potentially code in rules so that there is never less than 2 items in the second column. (@klepas I probably should've clarified that that might mean there will be cases where there is ONLY one column)

@petronbot I know you said CSS can do automatic even-dispersing between columns, but can it be told when it is okay to put it all in a single column?

**Comment by klepas** [comment]
Here’s two updated screengrabs.

**Desktop**, everything is OK, unless we have odd items:

jt-toc-desktop-even-margin-leak

**Mobile**, where we have some overlapping of the `article` `h1` and the `aside` `nav`:

jt-toc-mbl-nav-article-h1-overlap

**Comment by klepas** [comment]
This is what we look like when we go to a single column, just for reference I suppose.

toc-singlecol-desktop

I haven’t fixed the mbl view overlapping bug yet.

aside-nav-mbl-overlapping

**Comment by hannah-ustwo** [comment]
Related but different - can we hide the jump-to bit when a page only has ONE section? i.e. the current buttons page? Seems a bit unnecessary to have at least in a case like this.
@klepas @petronbot 

screen shot 2016-07-07 at 4 45 32 pm

**Comment by klepas** [comment]
LGTM. Nice.

File: assets/sass/_navigation.scss [hunk 1]
```diff
@@ -216,47 +216,47 @@ Style guide: Navigation.3 Skip Links
...
-        column-count: 2;
-        column-width: 40%;
-        column-gap: $gutter * 4;
-      }
+Used to group a collection of text links, Index links can appear at the beginning of an article page as a table of contents, or as a group of links to related content.
 
-      li {
-        margin-bottom: $medium-spacing;
-        font-weight: $bold-font-weight;
+An optional heading is included in the markup example. Any level leading from `h2` to `h6` can be used. Use a heading level that's appropriate for your document outline.
```
Comment (klepas) [inline assets/sass/_navigation.scss:225]: I like this, a lot. Nice addition — I saw the code that reflected this and was confused at first, but it makes total sense.

**Comment by klepas** [comment]
Final review done: looks great. Thanks @petronbot.

**Pull request merged**